packages/compass-query-bar/src/components/generative-ai/query-feedback.tsx

```diff
@@ -0,0 +1,118 @@
+import {
+  Button,
+  Icon,
+  FeedbackPopover,
+  css,
+  spacing,
+  cx,
+  palette,
+  useDarkMode,
+} from '@mongodb-js/compass-components';
+import React, { useRef, useState } from 'react';
+import { connect } from 'react-redux';
+
+import { submitFeedback } from '../../stores/ai-query-reducer';
+
+const suggestionActionButtonStyles = css({
+  flexShrink: 0,
+  display: 'flex',
+  gap: spacing[2],
+});
+
+const feedbackButtonLightStyles = css({
+  // We fill the icon colors here as there is a bug with the
+  // LeafyGreen ThumbsUp and ThumbsDown icons.
```

Comment:
Is there a ticket we can link to here?

Reply:
Added one! Didn't have at the time of writing, thanks for catching.